Facepunch.Steamworks/SteamUtils.cs

```diff
@@ -261,5 +261,20 @@ internal static bool IsCallComplete( SteamAPICall_t call, out bool failed )
 		/// Returns whether this steam client is a Steam China specific client, vs the global client
 		/// </summary>
 		public static bool IsSteamChinaLauncher => Internal.IsSteamChinaLauncher();
+
+		/// <summary>
+		/// Initializes text filtering, loading dictionaries for the language the game is running in.
+		/// Users can customize the text filter behavior in their Steam Account preferences
+		/// </summary>
+		public static bool InitFilterText( uint unFilterOptions ) => Internal.InitFilterText( unFilterOptions );
```

Comment:
Not sure I like this unFilterOptions

Reply:
If I look the steamworks API it is not present, however here in the native interface it is present. I think I can remove the parameter and just pass 0;

Reply:
I found a reference in the Steamwork.NET source:
https://github.com/rlabrecque/Steamworks.NET/blob/7d5c06394235361c5377f1b81c320218d74c7a50/Plugins/Steamworks.NET/autogen/isteamutils.cs#L290
Saying it not used and should be set to 0, I will do this as well.